src/Plugin/os2web/DataLookup/DatafordelerCVR.php

```diff
@@ -64,19 +64,23 @@ public function submitConfigurationForm(array &$form, FormStateInterface $form_s
   /**
    * {@inheritdoc}
    */
-  public function lookup($cvr) {
+  public function lookup(string $param): CompanyLookupResult {
     try {
-      $response = $this->httpClient->get('hentVirksomhedMedCVRNummer', ['query' => ['pCVRNummer' => $cvr]]);
+      $msg = sprintf('Hent virksomhed med CVRNummer: %s', $param);
```

Comment:
Consider renaming to `$auditMessage`, say, to make it clear that the message is used for (and only for) auditing.

Reply:
It is used on the next line....